megamek/src/megamek/common/Infantry.java

```diff
@@ -1577,75 +1576,79 @@ else if(nSneak == 3) {
         //Cost of armor on a per man basis added
         cost += (armorcost * menStarting);
 
+        // Price multiplier includes anti-mech training, motive type, and specializations
+        cost = cost * getPriceMultiplier();
+
+        //add in field gun costs
+        for (Mounted mounted : getEquipment()) {
+            if(mounted.getLocation() == LOC_FIELD_GUNS) {
+                cost += Math.floor(mounted.getType().getCost(this, false, mounted.getLocation()));
+            }
+        }
+        return cost;
+    }
+
+    @Override
+    public double getPriceMultiplier() {
+        double priceMultiplier = 1.0;
 
         //Anti-Mek Trained Multiplier
         if (isAntiMekTrained()) {
-            multiplier = 1;
+            priceMultiplier *= 5.0;
```

Comment:
Why 5x for anti-mek trained? Previous code was 1x (unless this diff is odd).

Reply:
Oh sorry, I should have called that out in the commit. 5.0x is the correct multiplier per the Tech Manual (p. 282).

It looks like this flag might not actually be getting used though - I see no cost difference between the regular and Anti-Mech foot platoons with this change applied.

Reply:
TM page 282 shows anti-mech training is x5 price multiplier

Reply:
@RAldrich that'll be an existing bug then.